tests/expose_test.go

```diff
@@ -727,7 +727,7 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				job = runHelloWorldJob(serviceIP, servicePort, vm.Namespace)
 
 				By("Waiting for the job to report a failed connection attempt.")
-				Expect(tests.WaitForJobToFail(job, 120*time.Second)).To(Succeed())
+				Expect(tests.WaitForJobToFail(job, 240*time.Second)).To(Succeed())
```

Comment:
I feel waiting 4 minutes for a test client to get a response is not a good thing; this could mask other issues.

Does the PR work fine without this ? I'd prefer if the timeout stays unchanged.

Reply:
Nope, the test fails because of the increased ping timeout, we need to wait more for checking that the job actually failed. In this case I think it is safe, we are expecting a failure connecting to an already deleted VMI.

Reply:
Right. Thanks for pointing out the scenario. Lazy me didn't check the scope of the test - just saw a timeout being doubled.